Connect4.js/src/app/app.ts

```diff
@@ -0,0 +1,54 @@
+import 'source-map-support/register'
+import Connect4Client, { GameState, IGame } from './connect4-client'
+
+const teamName = "CoffeeZero";
+const teamPassword = "1234321";
+
+const connect4Client = new Connect4Client();
+
+function playGame(playerId: string) {
+  connect4Client.getGame(playerId)
+    .then(x => {
+      const game = x.body;
+      switch (game.CurrentState) {
+        case GameState.RedWon:
+          console.log(game.RedPlayerID === playerId ? "You Won" : "You Lost");
+          break;
+
+        case GameState.YellowWon:
+          console.log(game.YellowPlayerID === playerId ? "You Won" : "You Lost");
+          break;
+
+        case GameState.RedToPlay:
+          if (game.RedPlayerID === playerId) {
+            makeMove(game, playerId)
```

Comment:
Just noticed the typing of the argument comes from the function definition anyway, so ignore this :D